tests/integration/globalSetup.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-env node */
+
+import TestContentServer from "../contents/server.mjs";
+
+export default () => {
+  // Start the server content prior to unit tests
+  const CONTENT_SERVER_PORT = 3000;
+  const testContentServer = TestContentServer(CONTENT_SERVER_PORT);
+};
```

Comment:
I don't get the point of that file compared to the `../contents/server.mjs`.

If the issue is that we need an exported function without parameters, maybe we should just rely on globals :/

Reply:
Yes the issue is that Vitest execute the default export function of the file specified in the vitest config : `globalSetup: "tests/integration/globalSetup.js"`

We cannot pass argument to that function. 
I can change `contents/server.mjs` so that it uses globals for the port number, but I don't understand why it would be better ?

Reply:
Just having one supplementary file, just to set a default argument and call it is seems too hacky

Maybe `TestContentServer` could just rely on a default port?

Also at worst it seems a `provide` API is available for the server to transmit data (like the port) to tests. Though in reality, I would prefer the server be separated from the testing framework and that it was the caller that provided the setup :/